### ISSUE TITLE
Allow launching alternate URLs

### ISSUE THREAD

**Issue by Aidoboy** [jiahaog/nativefier#380]
I think this could be really useful. For example: you have a youtube app, (side-note, why is the YouTube executable named `you-tube`? I just feel it looks weird.) and it normally launches "https://youtube.com". You want multiple shortcuts, say one that launchers "https://www.youtube.com/user/github" to get straight to the GitHub YouTube channel. What if you could run `you-tube https://www.youtube.com/user/github` or even just `you-tube user/github` to get another window that's still part of the same application?

**Comment by ronjouch** [comment]
@Aidoboy thx for the feedback.

Questions:

* Could you detail what use case / problem led you to this?
    * Put differently: what would this feature solve that simply creating a second nativefied app wouldn't?
* What do you mean by *"get another windows that's still part of the same application"*?
* What kind of associated behavior would you expect?

→ Given some thought to these questions, would https://github.com/jiahaog/nativefier/issues/348 - *"Support "shared runtime" mode, bundling apps as pure configuration"* be what you're looking for too? Is this fair to consider your bug a duplicate?

Comments:

* Regarding the weird inferred name, you can override it with option `--name`.

**Comment by Aidoboy** [comment]
No, I don't think this is a duplicate of #348, for several reasons.

- Each extra app has separate cookies, which is rather inconvenient. I'll probably start to get annoyed the 6th time I pull my Twitch password out of Keepass.
- I want the instances of the app to share the same icon on the taskbar, so if I had 3 Twitch windows open with separate streams that each launched from their prospective shortcuts, the taskbar would start to get really cluttered with icons that should really be stacked.

Is that clearer?

**Comment by Aidoboy** [comment]
@ronjouch Yeah, that re-title is good. In the meantime, do you think I could have multiple nativefier executables in the same directory with modified path names but the same `--name` names? Or maybe soft link cookie directories?

**Comment by ronjouch** [comment]
> *In the meantime, do you think I could have multiple nativefier executables in the same directory with modified path names but the same `--name` names? Or maybe soft link cookie directories?*

@Aidoboy try it, but ⚠️ I wouldn't be surprised to see the separate chrome instances at best stepping on each other toes, at worst corrupting the shared user profile / cookies.
